### DISCUSSION TITLE
Outlook Email Integration - Personal Account

### DISCUSSION THREAD

**Discussion by Byzza** [RogerSelwyn/O365-HomeAssistant#243] [Q&A] [answered]
Hi,
I'm trying to integrate our Families outlook.com calendar and looking at the documentation personal accounts should be possible, but when I go to create an Azure App, it comes up with "the ability to create applications outside of a directory has been depreciated.

without going through setting up a full Azure account, is there any way to get this working with a personal email address?

![image](https://github.com/user-attachments/assets/374509fb-edd5-45e8-9767-d6e4bf01e149)

**Comment by RogerSelwyn** [comment, marked as answer]
See this issue - https://github.com/RogerSelwyn/O365-HomeAssistant/issues/241
